docs/configuration.md

```diff
@@ -378,9 +402,33 @@ Permitted values are `error`, `warn`, `info` and `debug`. The default is
         disable: false
 
 The storage option is **required** and defines which storage backend is in use.
-You must configure one backend; if you configure more, the registry returns an error.
+You must configure one backend; if you configure more, the registry returns an error. You can choose any of these backend storage drivers:
+
+* The [`filesystem` storage backend](storage-drivers/filesystem.md) uses the local
+disk to store registry files. It is ideal for development and may be appropriate
+for some small-scale production applications.
+
+* The [`azure` storage backend ](storage-drivers/azure#parameters) uses Microsoft's Azure Blob Storage.
+
+* The [`gcs` storage backend](storage-drivers/gcs.md) uses Google Cloud Storage.
+
+* The [`rados` storage driver](storage-drivers/rados.md) uses Ceph Object Storage.
+
+* The [`s3` storage backend](storage-drivers/s3.md) uses Amazon's Simple Storage Service (S3).
 
-If you are deploying a registry on Windows, be aware that a Windows volume mounted from the host is not recommended. Instead, you can use a S3, or Azure, backing data-store. If you do use a Windows volume, you must ensure that the `PATH` to the mount point is within Windows' `MAX_PATH` limits (typically 255 characters). Failure to do so can result in the following error message:
+* The [`swift` storage backend](storage-drivers/swift.md) uses Openstack Swift object storage.
+
+* The [`oss` storage backend](storage-drivers/oss.md) uses Openstack Swift object storage.
+
+For purely tests purposes, you can use the [`inmemory` storage
+driver](storage-drivers/inmemory.md). **Never** use this option in a production
```

Comment:
This might be a little strong. This is not as bad as, say, silly auth. "This driver is not intended for production usage. If you would like to run a registry from volatile memory, use the `filesystem` driver on a ramdisk."